docs/api/storiesof.md

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
Was this intentionally re-added? I was thinking we were going to move this to a `ADVANCED.md` or something.